Dockerfile

```diff
@@ -3,6 +3,6 @@ MAINTAINER Graham Moore "graham.moore@sesam.io"
 COPY ./service /service
 WORKDIR /service
 RUN pip install -r requirements.txt
-EXPOSE 5000/tcp
+EXPOSE 5001/tcp
```

Comment:
why do you need to change the port here? would it not break the backward compatibility?

Reply:
I did it because the python program runs on port 5001, but I changed it back because _why change something weird which works_